spec/controllers/v0/profile/payment_history_controller_spec.rb

```diff
@@ -6,18 +6,32 @@
   let(:user) { create(:evss_user) }
 
   describe '#index' do
-    context 'with a valid bgs response' do
-      it 'returns true if a logged-in user has a valid va file number' do
+    context 'with only regular payments' do
+      it 'returns only payments and no return payments' do
```

Comment:
This is probably so edge-casey to not merit a test, but what about the case when there are only return payments?

Reply:
👍 That would be good to add. This PR is blocking Chika's mobile api PR (which is blocking FE) so I think I'll get this in and include that to the upcoming PRs for date filtering.